tile-reduce/index.d.ts

```diff
@@ -0,0 +1,139 @@
+// Type definitions for tile-reduce 3.1.1
+// Project: https://github.com/mapbox/tile-reduce
+// Definitions by: Denis Carriere <https://github.com/DenisCarriere>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace NodeJS  {
+  interface Global {
+    mapOptions: any
+  }
+}
+
+interface startEvent {
+    (): void;
+}
+
+interface mapEvent {
+    (tile: TileReduce.Tile, workerId: number): void;
+}
+
+interface reduceEvent {
+    (result: any, tile: TileReduce.Tile): void;
+}
+
+interface endEvent {
```

Comment:
Since this interface is only used once, you could just inline it with `callback: (error: any) => void`. Same for other `*Event` interfaces defined here.